src/OpenTelemetry.Instrumentation.AWSLambda/README.md

```diff
@@ -15,16 +15,52 @@ Add `AddAWSLambdaConfigurations()` to `TracerProvider`.
 ```csharp
 TracerProvider tracerProvider = Sdk.CreateTracerProviderBuilder()
         // add other instrumentations
-        .AddAWSLambdaConfigurations()
+        .AddAWSLambdaConfigurations(options => options.DisableAwsXRayContextExtraction = true)
         .Build();
 ```
 
+### AWSLambdaInstrumentationOptions
+
+`AWSLambdaInstrumentationOptions` contains various properties to configure
+AWS lambda instrumentation.
+
+- [`DisableAwsXRayContextExtraction`](/AWSLambdaInstrumentationOptions.cs#L25)
```

Comment:
Linking to a particular line seems dangerous, one would need to remember to update the readme for any change in this file. It will surely get out of sync. I would just link the whole file once, and not link each option. The radical approach would be to not even list them but just point to the file for the list, but one could also list them with a brief one-sentence summary.

Reply:
Right. I replaced both links by the file reference.